src/components/TablePreview/TablePreview.tsx

```diff
@@ -51,17 +52,23 @@ const TablePreview: FC<Props> = ({}) => {
                                     </td>
                                     <td style={name.style}>{name.textValue}</td>
                                     <td style={spacerStyle}></td>
-                                    <td style={phone.style}>{phone.textValue}</td>
+                                    <td style={phone.style}>
+                                        <a href={`tel:${phone.textValue}`}>{phone.textValue}</a>
+                                    </td>
                                 </tr>
                                 <tr>
                                     <td style={title.style}>{title.textValue}</td>
                                     <td style={spacerStyle}></td>
-                                    <td style={email.style}>{email.textValue} </td>
+                                    <td style={email.style}>
+                                        <a href={`mailto:${email.textValue}`}>{email.textValue}</a>
+                                    </td>
                                 </tr>
                                 <tr>
                                     <td style={company.style}>{company.textValue}</td>
                                     <td style={spacerStyle}></td>
-                                    <td style={linkedin.style}>{linkedin.textValue}</td>
+                                    <td style={linkedin.style}>
+                                        <a href={linkedin.textValue}>LinkedIn</a>
+                                    </td>
```

Comment:
Now that the word `LinkedIn` is hardcoded, will appear as soon as `hasData` is true, could we throw a ternary in here to fix it ? 
<img width="726" alt="image" src="https://github.com/evargast/signature-generator/assets/52969178/b1f11cf5-2df3-4e6d-a35c-84762585ea8a">

Reply:
Oh snap! on it!